meta-resin-common/recipes-support/resin-persistent-logs/resin-persistent-logs.bb

```diff
@@ -1,32 +1,23 @@
-DESCRIPTION = "Resin data partition expander"
+DESCRIPTION = "Resin persistent logs"
 LICENSE = "Apache-2.0"
 LIC_FILES_CHKSUM = "file://${RESIN_COREBASE}/COPYING.Apache-2.0;md5=89aea4e17d99a7cacdbeed46a0096b10"
 
 SRC_URI = " \
-    file://resin-data-expander \
-    file://resin-data-expander.service \
-    "
+    file://resin-persistent-logs \
+    file://resin-persistent-logs.service"
```

Comment:
In such lists we usually leave the '"' to be on the last line alone. Shall we do the same here for consistency of style?